sandbox/registration-processor-mz.properties

```diff
@@ -908,6 +920,16 @@ registration.processor.validateFile=true
 registration.processor.validateChecksum=true
 
 registration.processor.applicant.type=applicanttype-document-mapping.json
+registration.processor.validateApplicantDocument.process=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
+registration-processor.validatemandotary.process=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
+registration.processor.validateMasterData.process=NEW,UPDATE,LOST,BIOMETRIC_CORRECTION
+registration.processor.validateIndividualBiometricsforSubProcess.enabled=true
+registration-processor.validateOfficerOrSupervisorBiometricsforSubProcess.enabled=true
+registration.processor.validateMachineforSubProcess.enabled=true
+registration-processor.validateRegistrationCenterforSubProcess.enabled=true
+registration-processor.validateUMCMappingforSubProcess.enabled=true
+registration-processor.validateDeviceforSubProcess.enabled=true
+registration-processor.validateCenterIdAndTimestampForSubProcess.enabled=true
```

Comment:
1. Move all these properties to packet validator stage section after the below property
registration.processor.validateApplicantDocument=true (I think even this property to be removed) (around line 473)
2. Make all the property to start with "mosip.regproc.packet.validator." instead of "registration.processor."
3. Change all the properties taking comma separated to end with processes (plural)
4. We cannot use camel case in the properties, change all multi words to hyphen separated
5. Not sure if all these properties belong to validator stage, if not move it to the end of each stage sections in the properties file
6. No properties should take enabled true or false generally for the subprocess, everyone should take a comma separated processes for which that validation is to be done.
7. Please add comments above each property to explain the behaviour change this property holds

Reply:
Un used subprocesses  and other properties are removed,registration.processor.validateApplicantDocument this is still used in code so kept.
camel case updated with hyphen.
Comments added.Properties are moved.

Reply:
resolved